### PR TITLE
[cli] Use `getArgs()` instead of `mri` to parse arguments in `vc projects`

### PR DESCRIPTION
**Pull request by TooTallNate**
* Ensures that `vc --debug projects ls` properly prints the Projects listing, rather than the usage help info.
 * `vc projects` (without a subcommand) shows the Projects listing (this is consistent with i.e. `vc domains`).
 * Returns with exit code `2` when the usage help info is printed (standard Unix convention).

### CODE REVIEW

**Review by styfle — COMMENTED** [review]
Needs a CLI test

File: packages/now-cli/test/integration.js [hunk 1]
```diff
@@ -2916,3 +2916,19 @@ test('reject conflicting `vercel.json` and `now.json` files', async t => {
     formatOutput({ stderr, stdout })
   );
 });
+
+test('`vc --debug project ls` should output the projects listing', async t => {
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [...defaultArgs, '--debug', 'project', 'ls'],
```
Comment (styfle) [inline packages/now-cli/test/integration.js:2923]: I'm not sure if it will make a difference, but usually defaultArgs is at the end

**Review by styfle — APPROVED** [review]

**Comment by TooTallNate** [comment]
Added a test case.

File: packages/now-cli/test/integration.js (hunk 1 above)
Comment (TooTallNate) [reply packages/now-cli/test/integration.js:2923]: Good point, but I don't think it makes a difference in this case (the issue was happening when `--debug` came before `project`). This test was copied from the test above.

**Review by AndyBitz — APPROVED** [review]

**Pull request merged**